sdk/core/azure-core/src/environment_log_level_listener.cpp

```diff
@@ -120,11 +120,30 @@ EnvironmentLogLevelListener::GetLogListener()
 
   static std::function<void(Logger::Level level, std::string const& message)> const consoleLogger =
       [](auto level, auto message) {
-        std::cerr << '['
-                  << Azure::DateTime(std::chrono::system_clock::now())
-                         .ToString(
-                             DateTime::DateFormat::Rfc3339, DateTime::TimeFractionFormat::AllDigits)
-                  << "] " << LogLevelToConsoleString(level) << " : " << message << std::endl;
```

Comment:
Now all the messages that do not use stream, no longer printed from a new line, I very much disagree with that.

Reply:
???? THat is not my experience at all.

If there is no crlf at the end of the message, we use std::endl. If there is a crlf at the end of the message, we just flush the stream.

There are explicit tests to verify that this functionality works as expected.